keyboards/handwired/3dp660_oled/3dp660_oled.c

```diff
@@ -0,0 +1,121 @@
+// Copyright 2023 xia0 (@xia0)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#if defined (OLED_ENABLE) && ! defined (OLED_USER_DEFINED)
```

Comment:
```suggestion
#if defined (OLED_ENABLE)
```

Reply:
users can define this value to specify oled code within their keymap

Reply:
That's the purpose of the `oled_task_user` function, which is to be used at the keymap level and override the `oled_task_kb` function.